system/jhtree/keybuild.cpp

```diff
@@ -402,6 +404,11 @@ class CKeyBuilder : public CKeyBuilderBase, implements IKeyBuilder
             activeNode = new CWriteNode(nextPos, keyHdr, true);
             nextPos += keyHdr->getNodeSize();
         }
+        if (NULL != activeNode->getLastKeyValue())
```

Comment:
picky: new code should use nullptr.  (The comparison against null could be removed)